Rewrote get started for new onboarding android/iOS (#1264)
home-assistant/companion.home-assistant: docs/getting_started/index.mdx

```yaml
# Recommended: includes mobile_app and most common integrations
default_config:

# If you've disabled default_config, you need at least:
# mobile_app:

# Optional integrations for additional features:

# Secure remote access via Home Assistant Cloud subscription
# cloud:

# Advanced iOS notifications (actionable notifications, categories)
# ios:
```

Learn more about [Home Assistant Cloud](https://support.nabucasa.com/hc/en-us/categories/24734619902749) and [remote access](https://support.nabucasa.com/hc/en-us/sections/26496346499997).

## Setting Up

You can set up the Companion App from any location if your Home Assistant instance is configured for [remote access](https://www.home-assistant.io/docs/configuration/remote/). If you're on the same network as your Home Assistant, the app will automatically detect it.

:::info
Always log in using your Home Assistant credentials. If you have [Trusted Networks](https://www.home-assistant.io/docs/authentication/providers/#trusted-networks) enabled, avoid using it for the app, otherwise the app will only work on the trusted network.
:::

### Step-by-step setup

> 1. **Download the app**: Get the [Home Assistant Companion App](/download).

Reviewer: ```suggestion
1. **Download the app**: Get the [Home Assistant Companion app](/download).
```

> 2. **Welcome screen**: Open the app and select **Connect to my Home Assistant** to begin setup.

Reviewer: Updated to match text I'm seeing in the Android app itself

```suggestion
2. **Welcome screen**: Open the app and select **Connect to my Home Assistant server** to begin setup.
```

3. **Network discovery**: The app searches for Home Assistant instances on your network.

   <details>
   <summary>**Server found automatically**</summary>

   If a single Home Assistant instance is found, the app will automatically offer you to connect to it.

   <img alt="Home Assistant companion app showing a single discovered server ready to connect" src="/assets/OneServerFound.png" width="400"/>

   </details>

   <details>
   <summary>**Multiple servers found**</summary>

   If multiple Home Assistant instances are found, you will see a list to choose from. Tap the server you want to connect to.

   <img alt="Home Assistant companion app showing a list of multiple discovered servers to choose from" src="/assets/MultipleServerFound.png" width="400"/>

   </details>

   <details>
   <summary>**Server not found or connecting remotely**</summary>

   If no server is found, or you're connecting from outside your home network:

>    <div style={{display: 'flex', gap: '1rem', alignItems: 'flex-start'}}>

Reviewer: Try to avoid HTML if possible. Can't this be solved with simply two images side by side, or a simple Markdown table?

Reviewer: Fair enough, the comment was "try to avoid". It seems to scale up/down correctly 👍

     <img alt="Home Assistant companion  app showing no servers found with Enter address manually button" src="/assets/NoServerFound.png" style={{height: '400px', width: 'auto'}}/>
     <img alt="Home Assistant companion app manual server entry screen with URL input field" src="/assets/ManualServer.png" style={{height: '400px', width: 'auto'}}/>
   </div>

   1. Tap **Enter address manually**
   2. Enter your Home Assistant URL

   :::tip
   Using [Home Assistant Cloud](https://www.nabucasa.com/config/remote/) Remote UI is recommended for secure remote access.
   :::

   </details>

4. **Login**: Enter your Home Assistant credentials to authenticate.

> 5. **Device naming**: Choose a name for your device as it will appear in Home Assistant. This name identifies your phone in automations and services.

Reviewer: - Entities also use the device name
 - Services was renamed to actions a while ago

```suggestion
5. **Device naming**: Choose a name for your device as it will appear in Home Assistant. This name identifies your device entities and in automations and actions.
```

   - Select **Allow Once**, **Allow While Using the App**, or **Don't Allow**
   - For full functionality, including background automations, choose **Allow While Using the App**, then in the next prompt **Allow always**   
   </details>

   :::warning
   If you deny location permission, the app will not create a `device_tracker` entity or location-based sensors for your device.
   :::

7. **Connection security level**: If using a non-encrypted URL (such as a local IP address), choose a security level:
   - **Most secure**: Only allows non-encrypted connections when you're on your home network (requires location permission)
   - **Less secure**: Allows non-encrypted connections from any network (not recommended for public networks)
   - Learn more about [connection security levels](/docs/getting_started/connection-security-level)

8. **Set up your home network** (Most secure only): If you selected **Most secure** in the previous step, you will be asked to confirm your home network.

    - The app detects your current Wi-Fi network name (SSID) and pre-fills it
    - This allows the app to know when you're home and can safely use non-encrypted connections

>     - You can add additional trusted networks later from the settings.

Reviewer 2: In android you have more options than just SSID right? Perhaps add them here?

9. **Setup complete**: The app finalizes the connection and takes you to your Home Assistant dashboard.

:::note
If you have difficulties setting up the app, refer to the [troubleshooting guide](/docs/troubleshooting/setup.md).
:::

### Notifications

After setup, you will be asked to enable notifications. This allows Home Assistant to send alerts to your device.

- **Standard notifications**: Regular alerts for automations, reminders, and updates
- ![iOS](/assets/iOS.svg) **Critical notifications**: High-priority alerts that play a sound even when your device is in silent mode or Do Not Disturb is enabled

Learn more about how to [send notifications](/docs/notifications/basic.md) from Home Assistant.

## Adding Additional Servers

![iOS](/assets/iOS.svg) ![Android](/assets/android.svg)

> 

Reviewer: If it applies to both we don't need to repeat this

```suggestion
```
